main.html

```diff
@@ -5,18 +5,106 @@
 		<meta charset="UTF-8">
 		<title>pokedex</title>
 		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<!-- make sure to link to your css file -->
+		<link rel="icon" href="assets/icons/pokeball_icon.png" type="image/x-icon" />
+		<link rel="stylesheet" type="text/css" href="main.css">
 	</head>
 
-	<body>
-		<!-- include a header tag -->
+	<body background="assets/backgrounds/background_home.png">
+		<header>
+			
+			<div class="dropDown">
+				<button type="button">Dropdown</button>
+			</div>
 
-		<!-- depending on how you structure your html, you should have 3 section tags -->
+			<div>
+				<h1>Pokedex</h1>
+			</div>
+
+			<div>
+				<img src="assets/icons/pokeball_icon.png">
+			</div>
+			
+		</header>
+
+<!-- depending on how you structure your html, you should have 3 section tags -->
 			<!-- 1) search box -->
 			<!-- 2) selected pokemon: picture, property and description of pikachu -->
 			<!-- 3) other pokemon pictures in 2x6 format -->
+		<section>
+			<input id="searchBar" type="text" placeholder="Search Pokemon">
+		</section>
 
-		<!-- include navigation buttons -->
+		<section id="Pokemon info">
+
+			<figure>
+
+				<div>
+					<img src="assets/images/pikachu.png" alt"pikachu">
+				</div>
+				<div>
+					<figcaption>No.001 Pikachu</figcaption>
```

Comment:
This div extends the entire height of the image. As a result you see empty space after your list items. During such issues always look out for ways to style or orient these components such that they are dynamic based on the content.